AirCasting/SearchAndFollow/SearchMap/SearchMapView.swift

```diff
@@ -154,7 +176,10 @@ private extension SearchMapView {
                         .onMarkerChange(action: { pointer in
                             viewModel.markerSelectionChanged(using: pointer)
                         })
-                        .border((viewModel.cardPointerID.number == session.id ? Color.accentColor : .clear), width: 1)
+                        .overlay(
+                            RoundedRectangle(cornerRadius: 8)
+                                .stroke((viewModel.cardPointerID.number == session.id ? Color.accentColor : .clear), lineWidth: 1)
```

Comment:
shouldn't this be done in a viewmodel way of things instead of doing the logic here? Like:
```Swift
.stroke(viewModel.strokeColor, lineWidth: 1)
```